selfdrive/car/tests/test_models.py

```diff
@@ -39,6 +42,53 @@
 MAX_EXAMPLES = int(os.environ.get("MAX_EXAMPLES", "300"))
 CI = os.environ.get("CI", None) is not None
 
+@st.composite
+def car_control_strategy(draw):
```

Comment:
you need to have openpilot generate these CAN messages, we want to ensure that panda doesn't *block* anything openpilot will try to send to start

Reply:
oh ok so it should somehow get those CAN messages from `state_control` in the `Control` class from controlsd.py with `CarStates` obtained with fuzzed data
it would require some other processes to run along it or i'm missing something